src/app/sponsor/our-partners/page.js

```diff
@@ -0,0 +1,161 @@
+    {
+      image: GDCBg,
+      title: "Google Developer Groups on Campus: UGA",
+      description:
+        "GDGC UGA co-hosts their meetings with DevDogs, getting you in the Google sphere.",
+      link: "https://gdg.community.dev/gdg-on-campus-university-of-georgia-athens-united-states/",
+    },
+    {
+      image: KTPBg,
+      title: "Kappa Theta Pi: Phi Colony",
+      description:
+        "KTP is UGA's first technology frat, developing the next generation of industry professionals",
+      link: "https://ktpgeorgia.com/",
+    },
+    {
+      image: ACMBg,
+      title: "ACM @ UGA",
+      description:
+        "Every week, ACM hosts workshops discussing the latest technologies.",
+      link: "https://acm-uga.github.io/",
+    },
+    {
+      image: EPIBg,
+      title: "Computer Science Career Initiative",
+      description:
+        "CSCI provides resources to assist students in their pursuit of a many technical fields.",
+      link: "https://uga.campuslabs.com/engage/organization/csci",
+    },
+    {
+      image: SOEBg,
+      title: "Society of Entrepreneurs",
+      description:
+        "Learn from and connect with founders and business-savvy individuals at SOE's many events.",
+      link: "https://www.soeuga.com/",
+    },
+    {
+      image: UGAHacks,
+      title: "UGA Hacks",
+      description:
+        "Every spring, UGA Hacks puts on a 3-day, 800+ person hackathon at MLC.",
+      link: "https://ugahacks.com/",
+    },
+  ];
+  return (
+    <div className="w-full section ">
+      <PageTitleTemplate
+        redText="Partners"
+        blackText="Campus "
+        reverse="true"
+      />
+      <p className="text-center font-bold text-[1.8rem] pt-[8rem]">
+        DevDogs is proud to be affiliated with
+      </p>
+      <div className="w-full flex flex-col items-center flex-nowrap justify-center page-main-side-padding mt-[2rem]">
+        <Link
+          href="https://gdg.community.dev/gdg-on-campus-university-of-georgia-athens-united-states/"
+          target="_blank"
+        >
+          <Image
+            src={GDC}
+            alt="Google Developer Group"
+            className="w-full max-w-[400px] sm:max-w-[500px] md:max-w-[800px] object-contain"
+          />
+        </Link>
+        <p className="text-center text-[1rem] sm:text-[1.5rem] page-main-side-padding mt-[2rem]">
+          DevDogs co-hosts its meetings with Google Developer Groups on Campus:
+          UGA, providing Google recognition to contributors, merchandise, and a
+          network of incredible people just like you.
+        </p>
+      </div>
+
+      <div className="text-center text-[2rem] sm:text-[2.5rem] mt-[4rem]">
+        <h2 className="font-[700] inline  text-BulldogRed">Campus </h2>
+        <h2 className="font-[700] inline text-MidnightBlue">Affiliates</h2>
+      </div>
+      <EmblaCarousel
+        slides={imagesForCarousel}
+        banner={true}
+        options={OPTIONS}
+      />
+
```

Comment:
The space between the carousel and the campus partners' images can be shorter in mobile view.

Mobile view:
![image](https://github.com/user-attachments/assets/d66d3577-a11c-4d4a-a970-fae47eee989e)